### PR TITLE
Add Leetcode 102

### PR DESCRIPTION
**Pull request by Tina66**

### CODE REVIEW

File: src/102_binary_tree_level_order_traversal.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ */
+class Solution {
+public:
+    vector<vector<int>> levelOrder(TreeNode* root) {
+        vector<vector<int>> res;
+        traverse(root, 1, res);
+        return res;
+    }
+    
+    void traverse(TreeNode *root, size_t level, vector<vector<int>> &res) {
```
Comment (jovan-wong) [inline src/102_binary_tree_level_order_traversal.md]: Is this function given by leetcode? If it's not, I think we should not use a reference for a output parameter.

File: src/102_binary_tree_level_order_traversal.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+**Submissions**
+
+* Success
+* Details:  
+Runtime: 4 ms, faster than 98.84% of C++ online submissions for Binary Tree Level Order Traversal.
+
+### Approach 2 :  Iteration
+
+* Time complexity O(n)
+* Space complexity O(1)
```
Comment (jovan-wong) [inline src/102_binary_tree_level_order_traversal.md]: Actually, I saw this code's space complexity is O(n).

File: src/102_binary_tree_level_order_traversal.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+```c++
+class Solution {
+public:
+    vector<vector<int>> levelOrder(TreeNode* root) {
+       vector<vector<int>> res;
+        if (nullptr == root) {
+            return res;
+        }
+        queue<TreeNode*> cur, next;
```
Comment (jovan-wong) [inline src/102_binary_tree_level_order_traversal.md]: Declare one variable at one line.

File: src/102_binary_tree_level_order_traversal.md (hunk 2 above)
Comment (Tina66) [reply src/102_binary_tree_level_order_traversal.md]: I see...yeah,It is O(n)

File: src/102_binary_tree_level_order_traversal.md (hunk 1 above)
Comment (Tina66) [reply src/102_binary_tree_level_order_traversal.md]: The function is given by this way.

**Pull request merged**